omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ObsController1_9Test.java

```diff
@@ -84,6 +87,38 @@ public void shouldGetAll() throws Exception {
 	}
 	
 	@Test
+	public void shouldGetallIncludingVoidedObs() throws Exception {
+		
+		executeDataSet("encounterWithObsGroup1_9.xml");
+		
+		MockHttpServletRequest getAllNonVoidedObsRequest = newGetRequest(getURI());
+		getAllNonVoidedObsRequest.addParameter("encounter", "62967e68-96bb-11e0-8d6b-9b9415a91465");
+		MockHttpServletResponse AllNonVoidedObsResponse = handle(getAllNonVoidedObsRequest);
+		List<Object> AllNonVoidedObsList = deserialize(AllNonVoidedObsResponse).get("results");
+		
+		assertEquals(AllNonVoidedObsList.size(), 6);
+		
+		MockHttpServletRequest deleteRequest = newDeleteRequest(getURI() + "/" + "11de743c-96cd-11e0-8d6b-9b9415a91465");
+		deleteRequest.addParameter("reason", "test voided obs");
+		handle(deleteRequest);
+		
+		MockHttpServletRequest getAllObsIncludingVoidedRequest = newGetRequest(getURI());
+		getAllObsIncludingVoidedRequest.addParameter("encounter", "62967e68-96bb-11e0-8d6b-9b9415a91465");
+		getAllObsIncludingVoidedRequest.addParameter("includeAll", "true");
+		MockHttpServletResponse allObsIncludingVoidedResponse = handle(getAllObsIncludingVoidedRequest);
+		List<Object> allObsIncludingVoidedList = deserialize(allObsIncludingVoidedResponse).get("results");
+		
+		assertEquals(allObsIncludingVoidedList.size(), 6);
+		
+		MockHttpServletRequest getAllNonVoidedObsRequest2 = newGetRequest(getURI());
+		getAllNonVoidedObsRequest2.addParameter("encounter", "62967e68-96bb-11e0-8d6b-9b9415a91465");
+		MockHttpServletResponse AllNonVoidedObsResponse2 = handle(getAllNonVoidedObsRequest2);
```

Comment:
Did you look at our conventions for variable names? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions